skiplang/skjson/ts/src/skjson.ts

```diff
@@ -12,7 +12,17 @@ import type {
 import { sk_isObjectProxy } from "@skip-wasm/std";
 import type * as Internal from "./skjson_internal_types.js";
 
+/* Redefining Skip runtime types/symbols to avoid circular module dependency */
 const sk_frozen: unique symbol = Symbol.for("Skip.frozen");
+type Constant = { [sk_frozen]: true };
+function sk_freeze<T extends object>(x: T): T & Constant {
+  return Object.defineProperty(x, sk_frozen, {
+    enumerable: false,
+    writable: false,
+    value: true,
+  }) as T & Constant;
+}
+/* END Skip runtime redefinitions */
```

Comment:
This is nice, thanks. When doing such things, I also put a comment on the other copy of the definition, so that people changing it have a chance to notice that they should change the copy as well.